AutoGraphTests/Stub.swift

```diff
@@ -112,3 +129,102 @@ extension String {
         return components.filter { !$0.isEmpty }.joined(separator: " ")
     }
 }
+
+extension InputStream {
```

Comment:
This is needed for https://github.com/remind101/AutoGraph/pull/84/files#diff-9ed08f48421332f9614168f59bb22ce6R84

Unfortunately there's a bug with URLProtocol where by the time the request is handed off to you the data is already in `httpBodyStream` and `httpBody` is set to `nil`. So we use this to read the body back out into `Data`.